admin_guide/manage_rbac.adoc

```diff
@@ -60,15 +60,15 @@ Authorization] section.
 ====
 
 [[viewing-cluster-roles]]
-=== Viewing Cluster Roles
+=== Viewing cluster roles
 ifdef::openshift-enterprise,openshift-origin[]
 To view the cluster roles and their associated rule sets:
 
 ----
 $ oc describe clusterrole.rbac
 ----
 
-.Viewing Cluster Roles
+.Viewing cluster roles
```

Comment:
I realize these were already there; but, are they needed? Seem superfluous.

Reply:
@mburke5678, I don't think so. I think that any xrefs can go to the section heading. (Does the team have a set practice for if or when we label commands and examples?)